template/Makefile

```diff
@@ -1,25 +1,58 @@
 # If you want to compile for arm64e, you'll need a macOS device or a arm64e device that's able to compile for arm64e.
-# Also, you'll have to remove '#import "KittyMemory/initializer_list"' from Menu.h for it being able to compile this menu.
 # Once done that, uncomment the "#arm64e" by removing the "#"
 
 # THEOS_DEVICE_IP = 127.0.0.1 -p 2222 # install to device from pc
-TARGET = iphone:latest:7.0
 ARCHS = arm64 #arm64e
-
 DEBUG = 0
 FINALPACKAGE = 1
 FOR_RELEASE = 1
 
+# 0 to treat warnings as errors, 1 otherwise.
+IGNORE_WARNINGS=0
+
+# only set this  to 1 if you are on mobile theos
+# assuming you have an sdk at your theos sdks directory
+# this will include c++ headers and other needed headers for your project so you don't need to manually include them or something like that
+# if some c++ headers are still missing in your sdk like "initializer_list" then manually copy them to your c++ headers directory and not your project folder
+# for example in my case c++ headers directory is located at /private/var/theos/sdks/iPhoneOS11.2.sdk/usr/include/c++/4.2.1/
+# please note, do not include c++ headers in your theos includes to enable c++ which is a ghetto solution and use this approach instead
+MOBILE_THEOS=1
+ifeq ($(MOBILE_THEOS),1)
+  # path to your sdk
+  SDK_PATH = /private/var/theos/sdks/iPhoneOS11.2.sdk/
```

Comment:
I think it's better to use: `$(THEOS)/sdks/iPhoneOS11.2.sdk/` here, as every installation guide creates this variable

Reply:
sure, I replaced it.

Reply:
Awesome, gonna merge it. Thanks again 👍